test/helpers/native-mocks.ts

```diff
@@ -1,51 +1,38 @@
-import * as React from "react";
+import { Component, ComponentClass, createElement, forwardRef, PropsWithChildren, ReactNode } from "react";
 import {
   Animated,
   MeasureInWindowOnSuccessCallback,
-  NativeMethods
+  NativeMethods,
 } from "react-native";
 
 import { MeasureOnSuccessCallbackParams } from "./measures";
 
-export function mockNativeComponent(modulePath: string, mockMethods: NativeMethods) {
```

Comment:
This helper function had to be refactored as it relied too much on `any` types, which caused the linter to heavily complain